cmd/root.go

```diff
@@ -45,6 +46,17 @@ func NewRootCmd(out io.Writer, args []string) *cobra.Command {
 	flags.Parse(args)
 	settings = new(EnvSettings)
 
+	// Helm plugins are not passed Helm's global flags but instead must
+	// read the global flag values from each corresponding environment variable.
+	// The below allows to use Helm's --kube-context global flag.
+	if ctx := os.Getenv("HELM_KUBECONTEXT"); ctx != "" {
+		settings.KubeContext = ctx
+	}
+
+	// Note that Helm's global --kubeconfig flag has a corresponding KUBECONFIG variable.
```

Comment:
Might read better as:
"Note that the plugin's --kubeconfig flag is set by the Helm plugin framework to the KUBECONFIG environment variable instead of being passed into the plugin."

Reply:
Done